apm-configuration/python.mdx

```diff
@@ -61,61 +66,67 @@ Refer more about Docker bridge networking [here](https://docs.docker.com/network
 kubectl get service --all-namespaces | grep mw-service
```

Comment:
"For Kubernetes, first identify the namespace where the MW Agent is running." above this command